.gitignore

```diff
@@ -31,6 +31,7 @@
 .idea/dynamic.xml
 .idea/uiDesigner.xml
 .idea/*
+my-remarks.txt
```

Comment:
I guess this was committed by accident 🙂

Reply:
yeah I have fixed it alreay.